doc/text.md

```diff
@@ -0,0 +1,90 @@
+# Text Rendering
+
+Flame has some dedicated classes to help you render text.
+
+# TextConfig
+
+A Text Config contains all typographical information required to render texts; i.e., font size and color, family, etc.
+
+Example usage:
+
+```dart
+const TextConfig config = TextConfig(fontSize: 48.0, fontFamily: 'Awesome Font', anchor: Anchor.rightBottom);
+```
+
+* fontFamily : a commonly available font, like Arial (default), or a custom font added in your pubspec (see [here](https://flutter.io/custom-fonts/) how to do it)
+* fontSize : font size, in pts (default `24.0`)
+* color : the color, as a `ui.Color` (see below, default black)
+
+To create a Color object, just pass in the color as an integer in the ARGB format.
+
+You can use Dart's hexadecimal notation to make it really easy; for instance: `0xFF00FF00` is fully opaque green (the 'mask' would be `0xAARRGGBB`).
+
+There is a color enum to make it easy to use common colors; it is in the material flutter package:
+
+```dart
+    import 'package:flutter/material.dart' as material;
+
+    Color color = material.Colors.black;
+```
```

Comment:
I like this explanation, but we could take the opportunity to link to a Color Palette documentation to instruct to better using colors and paints :)

Reply:
I am not very aware of the palette APIs, will merge this right now and we do the palette docs later. This way at least we remove the old and removed part of the text doc :D